python/lsst/ctrl/mpexec/cli/script/build.py

```diff
@@ -53,7 +62,14 @@ def build(  # type: ignore
         Path location for storing resulting pipeline definition in YAML format.
     show : `lsst.ctrl.mpexec.showInfo.ShowInfo`
         Descriptions of what to dump to stdout.
-    kwargs : `dict` [`str`, `str`]
+    butler_config : `str`, `dict`, or `lsst.daf.butler.Config`, optional
+        If `str`, `butler_config` is the path location of the gen3
+        butler/registry config file. If `dict`, `butler_config` is key value
+        pairs used to init or update the `lsst.daf.butler.Config` instance. If
+        `Config`, it is the object used to configure a Butler.
+        Only used to resolve pipeline graphs for --show pipeline-graph and
+        --show task-graph.
+    **kwargs
```

Comment:
The current version still has the type as `dict` `[`str`, `str`]`, but the change does not. Is there a reason for that?

Reply:
My understanding is that the right type for `**kwargs`  is the _value_ type only, since it's always a dict, and the keys are always strings.  And it wasn't actually true before that the values were all strings, and I think that's most appropriately handled in this case by leaving out the type from the docs entirely.